.storybook/preview.tsx

```diff
@@ -1,12 +1,20 @@
+import { CSSProperties } from 'react';
 import NextImage from 'next/image';
 import { withThemeByDataAttribute } from '@storybook/addon-themes';
 import { SiteProvider } from '../providers/siteProvider';
 import { LocaleProvider } from '../providers/localeProvider';
 import { OPEN_SANS_FONT, STORYBOOK_MODES, STORYBOOK_SIZES } from './constants';
 import type { Preview, ReactRenderer } from '@storybook/react';
+import * as Toast from '@radix-ui/react-toast';
```

Comment:
I think a dedicated Provider needs to be done even if it just simply wraps this on `providers/` folder as `NotificationProvider.tsx`

```tsx
import * as Toas from '@radix-ui/react-toast';

const viewportStyles: CSSProperties = {
  position: 'absolute',
  bottom: 0,
  right: 0,
};

export const NotificationProvider: FC<PropsWithChildren> = ({ children }) => (
  <Toast.Provider>
    {children}
    <Toast.Viewport style={viewportStyles} />
  </Toast.Provider>
);
```

This simply to keep the preview clean and the module reusable.

Reply:
updated